README.md

```diff
@@ -6,25 +6,25 @@ Library for building a Run on Slack Deno project. The artifacts produced from th
 
 _**Note:** The examples below use version `0.0.5` of `deno-slack-builder`; check the [Releases](https://github.com/slackapi/deno-slack-builder/releases) page and be sure to use the latest version._
 
-In a directory that contains a valid manifest file (`manifest.json` or `manifest.ts`), run the following:
+In a directory that contains a valid manifest file (`manifest.json`, `manifest.js`, or `manifest.ts`), run the following:
```

Comment:
Added `manifest.js` to the supported file types ([source](https://github.com/slackapi/deno-slack-builder/blob/main/src/manifest.ts#L25))